### PR TITLE
Add pixy

### PR DESCRIPTION
**Pull request by ksamuk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pixy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (marcelotrevisani) [inline recipes/pixy/meta.yaml]: What is the reason that you are skipping windows?
Is it a pure python project?

File: recipes/pixy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/pixy/meta.yaml]: You need to declare the entry points here

```suggestion
  number: 0
  entry_points:
    - pixy=pixy.py
```

File: recipes/pixy/build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/src/pixy.py $PREFIX/bin/pixy
+chmod +x $PREFIX/bin/pixy
```
Comment (marcelotrevisani) [inline recipes/pixy/build.sh]: Could you please delete this file and also the ``bld.dat`` I think they are not necessary

**Comment by marcelotrevisani** [comment]
Hi @ksamuk ,
I hope you don't mind. I modified a bit your recipe. If you have any questions regarding my modifications please feel free to ask. :)

**Comment by marcelotrevisani** [comment]
It is failing because the "scripts" option on your ``setup.py`` is wrong. I opened a PR there to fix it
Could you please integrate it and generate another release? That would be easy to do, otherwise we will need to patch this project

**Comment by marcelotrevisani** [comment]
https://github.com/ksamuk/pixy/pull/3

**Comment by marcelotrevisani** [comment]
I found another problem with the ``setup.py``, I will open another Pr in a few minutes

**Comment by ksamuk** [comment]
Ah, apologies! Thanks very much for looking into this, its greatly appreciated.

**Comment by marcelotrevisani** [comment]
> Ah, apologies! Thanks very much for looking into this, its greatly appreciated.

So, I just realized that... this project is a notebook right!? there are some stuff which you need to fix there, there are also some hardcoded paths such as
```python
# test data
# TBD: replace with parsed args
chromosome = "chrX"
vcf_path = 'data/vcf/ag1000/chrX_36Ag_allsites.vcf.gz'
zarr_path = 'data/vcf/ag1000/chrX_36Ag_allsites.zarr'
```

should it be like there? that looks a bit strange to me

**Comment by marcelotrevisani** [comment]
This project also has more dependencies such as ``pandas``, ``numpy``, ``allel``, ``zarr``

**Comment by marcelotrevisani** [comment]
Unfortunately, I cannot work on it
But as it is now, you should skip windows indeed, because there are some hardcoded paths which are using the unix separator ``/``

I did some improvements, but I am not able to continue any further for now. But I think you can continue from that

**Comment by ksamuk** [comment]
Great, I'll try to figure it out, thanks very much again for your help!

**Comment by ksamuk** [comment]
@marcelotrevisani I think we're good here, thanks again for your help with this!

**Comment by ksamuk** [comment]
ping @conda-forge/staged-recipes

File: recipes/pixy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+  entry_points:
+    - pixy = pixy.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (CJ-Wright) [inline recipes/pixy/meta.yaml]: I think you need to put this in as a skip, limitations on the python version only works with noarch.

File: recipes/pixy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: 40cb4db24d08d86347fe2a6dd64bd2e20f8a9c9671f19202086e5c72b670a776
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```
Comment (CJ-Wright) [inline recipes/pixy/meta.yaml]: you can combine these as `skip: True  # [win or py<3]`

Comment (ksamuk) [reply recipes/pixy/meta.yaml]: Thanks for your help with this!

**Comment by CJ-Wright** [comment]
This looks great! Thank you very much @ksamuk. Welcome to Conda-Forge!

**Comment by ksamuk** [comment]
Fantastic! Thanks very much @CJ-Wright and @marcelotrevisani for your patience and help!

**Pull request merged**